wiki/Legal/Copyright/Music_Licensing/en.md

```diff
@@ -0,0 +1,19 @@
+# Music Licensing
+
+The osu! team reaches out to artists that the community loves and new artists whose music is appropriate for beatmapping in order to form licensing agreements. Those who agree to osu!'s licensing terms are a part of the [Featured Artists](/wiki/Featured_Artists) program.
+
+Artists interested in joining the Featured Artist program are encouraged to contact osu! for licensing negotiations.
+
+## Terms
```

Comment:
missing info about usage of songs off-platform, like in livestreams or replay videos. whether it's covered or not, I think that's important to disclose here

Reply:
covered by line 11?

Reply:
i've addressed this with a refactor on line 11. i think @cl8n was more talking about non-official uploads off-platform, which we intentionally do not cover in our licensing.

Reply:
yep